lib/fastlane/plugin/wpmreleasetoolkit/actions/configure/configure_apply_action.rb

```diff
@@ -54,44 +54,42 @@ def self.prepare_repository
       ### Check with the user whether we should overwrite the file, if it exists
       ###
       def self.apply_file(file_reference, force)
-        begin
```

Comment:
The rest of that diff is indentation-related only; so despite the diff being large, the only change is the removal of the `begin` and `end` keywords which wrapped the whole function body ([method bodies are already implicit exception blocks](https://docs.rubocop.org/rubocop/1.0/cops_style.html#styleredundantbegin)).

Reply:
This is one of the reasons I love tools like RuboCop and SwiftLint: they actually teach you how to write _better_ code.